content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -161,7 +161,7 @@ Terraform Enterprise requires **at least one** of the following Docker Engine co
     runc --version
     ```
 
-1. If your Docker Engine and `runc` versions meet the requirements from previous steps, you're all set! Otherwise, proceed to [option 2](#).
+1. If your Docker Engine and `runc` versions meet the requirements from previous steps, you're all set! Otherwise, proceed to [option 2](#option-2-docker-engine-with-a-compatbile-libseccomp-version).
```

Comment:
```suggestion
1. If your Docker Engine and `runc` versions meet the requirements from previous steps, your system is properly configured. Otherwise, proceed to [option 2](#option-2-docker-engine-with-a-compatbile-libseccomp-version).
```